german/Leichen/Simulation/Simulation.md

```diff
@@ -8,4 +8,6 @@ Nachdem ich vergeblich auf k
 angebrachten Gegenst�nden - wie einer VR-Brille oder verkabelten Handschuhen - gewartet habe,
 muss ich mich nach anderen Hinweisen umschauen.
 
-Deshalb erw�ge ich folgende Optionen:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.